### DISCUSSION TITLE
Выводить в префильтре с селектом, в списке выбора только уникальные значения

### DISCUSSION THREAD

**Discussion by aityomin** [totumonline/totum-ru-issues-and-discussions#440] [Как это сделать?] [answered]
Дано - поле с неуникальными значениями. Например, фамилии.
Префильтр с селектом выдает список всех значений. Есть в таблице 20 Ивановых, он их все показывает. При выборе фильтрует конкретного Иванова.
Как сделать, чтобы он выводил только одного Иванова? И при этом фильтр работал :)

**Comment by erpic-ru** [comment, marked as answer]
В коде селекта для префильтра добавить

`bfield: 'familiya';`

Либо создать уникальный справочник фамилий и фильтровать по нему

**Reply by aityomin** [reply]
Спасибо! Однако с bfield фильтрация к сожалению не работает... Это было очевидным решением, сразу попробовали.

**Reply by aityomin** [reply]
Все оказалось просто! При использовании bfield в Name поля в строчной части нужно было вместо id поставить familiya.
